modules/rosa-sts-required-roles-and-policies.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: MODULE
+[id="rosa-sts-required-roles-and-policies_(context)"]
+= Required operator roles and policies
+
+Whether you are using the `rosa` command-line interface (CLI) tool or the ROSA guided installation in {cluster-manager-url} to create a cluster, you must create the proper operator roles as well as setting the correct OIDC provider. You may create these roles manually or through an automatic process that uses pre-defined names for these roles and polices.
```

Comment:
Suggestion: 
"...you must create the proper operator roles and set the correct OpenID Connect (OIDC) provider."

Reply:
s/may/can